.github/workflows/podvm.yaml

```diff
@@ -100,3 +117,4 @@ jobs:
           "UBUNTU_IMAGE_CHECKSUM="
           "BINARIES_IMG=${{ inputs.registry }}/podvm-binaries-${{ matrix.os }}-${{ matrix.arch }}:${{ inputs.binaries_img_tag }}"
           "BUILDER_IMG=${{ inputs.registry }}/podvm-builder-${{ matrix.os }}:${{ inputs.builder_img_tag }}"
+          "CAA_SRC_REF=${{ env.CAA_SRC_REF }}"
```

Comment:
note that there's also https://github.com/confidential-containers/cloud-api-adaptor/pull/1448 PR ongoing, IIUC it will allow you simplify this series

Reply:
yep, @jtumber-ibm has also sent another PR to avoid rebuilding the agent all the time. Plenty of work is on the queue to simplify/speed up the build process but this PR is considering the old way.